### ISSUE TITLE
Deal with scenario where supervisor isn't running

### ISSUE THREAD

**Issue by sujaymansingh** [EDITD/ansible-supervisor_task#12]
Sometimes (observed in a docker container) the role will fail.

```
TASK: [EDITD.supervisor_task | Update supervisor] *****************************
failed: [127.0.0.1] => {"changed": true, "cmd": ["supervisorctl", "update"], "delta": "0:00:00.073057", "end": "2016-08-10 14:12:29.548373", "rc": 2, "start": "2016-08-10 14:12:29.475316", "warnings": []}
stdout: error: <class socket.error>, [Errno 2] No such file or directory: file: /usr/lib/python2.7/socket.py line: 224
```

Looks like this is caused by supervisord not running when update is called: https://github.com/Supervisor/supervisor/issues/121#issuecomment-8571670

**Comment by sujaymansingh** [comment]
```
$ sudo service supervisor start
Starting supervisor: supervisord.
$ sudo service supervisor status
 is running
$ sudo supervisorctl update
$ sudo service supervisor stop
Stopping supervisor: supervisord.
$ sudo service supervisor status
 is  not running.
$ sudo supervisorctl update
error: <class 'socket.error'>, [Errno 2] No such file or directory: file: /usr/lib/python2.7/socket.py line: 224
$ sudo service supervisor start
Starting supervisor: supervisord.
$ sudo service supervisor status
 is running
$ sudo supervisorctl update
$ 
```

**Comment by sujaymansingh** [comment]
Closed by merging #13
